src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -115,15 +115,18 @@
    * Creates a new FirebaseUserManager instance.
    *
    * @param app A non-null {@link FirebaseApp}.
+   * @param tenantId The associated tenant ID if the user operations should be tenant-aware,
+   *     otherwise {@code null}
    */
-  FirebaseUserManager(@NonNull FirebaseApp app) {
+  FirebaseUserManager(@NonNull FirebaseApp app, @Nullable String tenantId) {
     checkNotNull(app, "FirebaseApp must not be null");
     String projectId = ImplFirebaseTrampolines.getProjectId(app);
     checkArgument(!Strings.isNullOrEmpty(projectId),
         "Project ID is required to access the auth service. Use a service account credential or "
             + "set the project ID explicitly via FirebaseOptions. Alternatively you can also "
             + "set the project ID via the GOOGLE_CLOUD_PROJECT environment variable.");
-    this.userMgtBaseUrl = String.format(ID_TOOLKIT_URL, "v1", projectId);
+    this.userMgtBaseUrl =
```

Comment:
I feel like a simple if-else will explain the logic here a bit better:

```
if (Strings.isNullOrEmpty(tenantId) {
  this.userMgtBaseUrl = ...
} else {
  this.userMgtBaseUrl = ...
}
```

Then in `getTenantUrlSuffix()` make tenantId non-null.

Reply:
Done.